src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/SideRail.razor.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bit.Client.Web.BlazorUI.Playground.Web.Models;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+
+namespace Bit.Client.Web.BlazorUI.Playground.Web.Pages.Components
+{
+    public partial class SideRail
+    {
+        [Inject] public IJSRuntime? JSRuntime { get; set; }
+
+        [Parameter] public List<SideRailParameter> SideRailParameters { get; set; }
+
+
+        private async Task ScrollToFragment(SideRailParameter targetElement)
```

Comment:
plz rename this method to `ScrollToItem` and also rename its argument to `targetItem`